models/articles-models.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection")
+
+const fetchArticleById = (article_id) => {
+    return db.query('SELECT * FROM articles WHERE article_id = $1;', [article_id])
+    .then((result) => {
+        return result.rows
```

Comment:
please send back the single article and not a single article nested within an array.

Reply:
tests will have to change slightly to accomodate this.